nova/virt/vmwareapi/cluster_util.py

```diff
@@ -143,7 +145,7 @@ def delete_vm_group(session, cluster, vm_group):
 
 
 @utils.synchronized('vmware-vm-group-policy')
-def update_placement(session, cluster, vm_ref, group_infos):
+def update_placement(session, cluster, vm_ref, group_infos, operation="add"):
```

Comment:
Looking at this function, I think I would refrain from touching it and add an explicit "remove_vm_from_group" function. Main reason is, that I currently cannot wrap my head around if we here skip the `if group_info.policies` in a valid way for only one use-case.
Also, the group might be empty after we remove the VM from it. It might need cleaning-up. But this is only done in the surrounding function. Anybody later using this function might miss it. Therefore, I think an additional function handling the removal case would be better.